src/test/java/org/opensearch/knn/indices/ModelGraveyardTests.java

```diff
@@ -79,6 +81,87 @@ public void testXContentBuilder() throws IOException {
         assertTrue(testModelGraveyard2.contains(testModelId2));
     }
 
+    // Validating {model_ids:[]}
+    public void testXContentBuilder2() throws IOException {
```

Comment:
Test names should be descriptive, not just enumerated.

One good method would be `test<Component>_when<Condition>_then<expectation>` if the same component is being tested in multiple tests.

Reply:
+1 to this comment. But I am not sure if we are following this across this repo. But if we want to start we should start doing it.

Reply:
but rather than using 2,3,4 in the names of the test cases better to put in the function name.